src/app/util/process-global-message.cpp

```diff
@@ -112,9 +111,7 @@ bool emAfProcessGlobalCommand(EmberAfClusterCommand * cmd)
 
     // If we are disabled then we can only respond to read or write commands
     // or identify cluster (see device enabled attr of basic cluster)
-    if (!emberAfIsDeviceEnabled(cmd->apsFrame->destinationEndpoint) && zclCmd != ZCL_READ_ATTRIBUTES_COMMAND_ID &&
-        zclCmd != ZCL_WRITE_ATTRIBUTES_COMMAND_ID && zclCmd != ZCL_WRITE_ATTRIBUTES_UNDIVIDED_COMMAND_ID &&
-        zclCmd != ZCL_WRITE_ATTRIBUTES_NO_RESPONSE_COMMAND_ID && clusterId != ZCL_IDENTIFY_CLUSTER_ID)
+    if (!emberAfIsDeviceEnabled(cmd->apsFrame->destinationEndpoint) && clusterId != ZCL_IDENTIFY_CLUSTER_ID)
```

Comment:
It is hard to follow the code that left and the comment now that the code that checks for `ZCL_WRITE_ATTRIBUTES_COMMAND_ID`, `ZCL_READ_ATTRIBUTES_COMMAND_ID` is removed.

Reply:
Changed removal to comment, and updated the comment above.